cmd/launcher/uninstall_linux.go

```diff
@@ -0,0 +1,59 @@
+//go:build linux
+// +build linux
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func removeLauncher(ctx context.Context, identifier string) error {
+	if identifier == "" {
+		identifier = "kolide-k2"
+	}
+
+	serviceName := fmt.Sprintf("launcher.%s", identifier)
+	packageName := fmt.Sprintf("launcher-%s", identifier)
+
+	// Stop launcher service
+	cmd := exec.CommandContext(ctx, "systemctl", []string{"stop", serviceName}...)
+	if err := cmd.Run(); err != nil {
+		return err
+	}
+
+	// Disable launcher service
+	cmd = exec.CommandContext(ctx, "systemctl", []string{"disable", serviceName}...)
+	if err := cmd.Run(); err != nil {
+		return err
+	}
+
+	// Tell the appropriate package manager to remove launcher
+	if _, err := os.Stat("/usr/bin/dpkg"); err == nil {
```

Comment:
what happens if both of these if conditions fail?

Reply:
Now this errors out